low-level/files/iec_mapping/sspl

```diff
@@ -0,0 +1,10 @@
+0050010001,Ipmitool,Ipmitool command execution error
+0050010002,Ipmitool,Ipmitool command executed successfully
```

Comment:
>Ipmitool command executed successfully

This message is for recovery right? It should state that then. How about below or something better ..

"ipmitool command execution success again" 

same applies to other tools

Reply:
Done.